src/main/java/uk/gov/hmcts/probate/transformer/WillLodgementCallbackResponseTransformer.java

```diff
@@ -94,6 +99,14 @@ public WillLodgementCallbackResponse addMatches(WillLodgementCallbackRequest req
         return transformResponse(responseCaseDataBuilder.build());
     }
 
+    public WillLodgementCallbackResponse addDocuments(WillLodgementCallbackRequest callbackRequest, List<Document> documents) {
```

Comment:
Could add a test in WillLodgementCallbackResponseTransformerTest, such as : 
shouldAddwillLodgementDepositReceiptToGeneratedDocuments